js/tji-datasets-explore.js

```diff
@@ -655,11 +671,11 @@ TJIChartView.prototype.create_chartview_DOM = function() {
     class: 'tji-chartview__record-count'
   });
 
-  this.ui.$download = jQuery('<button class="tji-btn-primary tji-chartview__download-button" disabled> <i class="fas fa-download"></i> Download</button>');
+  this.ui.$download = jQuery('<button class="tji-btn-primary tji-btn-download tji-chartview__download-button" disabled> <i class="fas fa-download"></i> Download</button>');
```

Comment:
We decided to put it in the buttons.scss file and match the button class names. The thought was that it shouldn't live with the Explore css because eventually we might want to use that class on download buttons on the homepage.